plugins/catalog-backend/src/ingestion/processors/CodeOwnersProcessor.ts

```diff
@@ -33,6 +34,11 @@ const ALLOWED_LOCATION_TYPES = [
   'gitlab/api',
 ];
 
+// TODO(Rugvip): We want to properly detect out repo provider, but for now it's
+//               best to wait for GitHub Apps to be properly introduced and see
+//               what kind of APIs that integrations will expose.
+const KNOWN_LOCATIONS = ['', '/docs', '/.bitbucket', '/.github', '/.gitlab'];
```

Comment:
FIXED IT 👌 

As mentioned in the comment I'm thinking we could hold off on any kind of integrations detection right now. `git-url-parse` doesn't provide enough information to know what the repo provider is.

With GitHub apps we'll likely end up with the requirement to be able to dynamically generate access tokens, as they only last for a short time. That'll likely lead us towards some kind of "access headers for URL"-type interface, where we might end up tagging along some type detection.